src/common/services/session/session.service.js

```diff
@@ -26,18 +28,35 @@ export const Sessions = {
   profile: 'cru-profile'
 }
 
+export const redirectingIndicator = 'redirectingFromOkta'
+export const locationOnLogin = 'locationOnLogin'
+export const locationSearchOnLogin = 'locationSearchOnLogin'
+export const checkoutSavedDataCookieName = 'checkoutSavedData'
+export const createAccountDataCookieName = 'createAccountData'
+export const cookieDomain = '.cru.org'
+
 export const SignInEvent = 'SessionSignedIn'
 export const SignOutEvent = 'SessionSignedOut'
+export const LoginOktaOnlyEvent = 'loginAsOktaOnlyUser'
 
-const session = /* @ngInject */ function ($cookies, $rootScope, $http, $timeout, envService) {
+const session = /* @ngInject */ function ($cookies, $rootScope, $http, $timeout, $window, $location, envService) {
   const session = {}
   const sessionSubject = new BehaviorSubject(session)
   let sessionTimeout
   const maximumTimeout = 30 * 1000
+  const authClient = new OktaAuth({
+    issuer: envService.read('oktaUrl'),
+    clientId: envService.read('oktaClientId'),
+    redirectUri: `${window.location.origin}/sign-in.html`,
```

Comment:
Does this client get created on branded checkout? Does the non-existence of this page on BCO affect anything?

Reply:
I don't believe BCO has a way for users to log in via Okta, so I don't think this will be an issue as this is the URI Okta returns the user to after a successful login.